config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml

```diff
@@ -0,0 +1,33 @@
+periodics:
+  - interval: 6h
+    name: check-dependency-stats-periodical
+    decorate: true
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
```

Comment:
This will be `master`, right? Confirming cause this might mean that initially we are checked into master instead of the PR commit.

Reply:
yes master is good